### PR TITLE
[update] add awschaos api

### PR DESCRIPTION
**Pull request by fewdan**
Signed-off-by: “fewdan” <fewdan@hotmail.com>

### What problem does this PR solve?
add awschaos api

### What is changed and how does it work?
add awschaos api in dashboard (mainly experiments). 

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by fewdan** [comment]
@g1eny0ung Added the awschaos api, maybe the front end also needs to be modified.

**Comment by g1eny0ung** [comment]
> @g1eny0ung Added the awschaos api, maybe the front end also needs to be modified.

I will review this PR ASAP and update the UI.

File: pkg/core/experiment.go [hunk 1]
```diff
@@ -157,14 +157,15 @@ func (s *SelectorInfo) ParseSelector() v1alpha1.SelectorSpec {
 
 // TargetInfo defines the information of target objects.
 type TargetInfo struct {
-	Kind         string            `json:"kind" binding:"required,oneof=PodChaos NetworkChaos IoChaos KernelChaos TimeChaos StressChaos DNSChaos"`
+	Kind         string            `json:"kind" binding:"required,oneof=PodChaos NetworkChaos IoChaos KernelChaos TimeChaos StressChaos DNSChaos Awschaos"`
```
Comment (g1eny0ung) [inline pkg/core/experiment.go:160]: `Awschaos` => `AwsChaos`.

**Review by g1eny0ung — CHANGES_REQUESTED** [review]

**Review by g1eny0ung — APPROVED** [review]
LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**